cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/InstanceStoreMetadata.java

```diff
@@ -5,25 +5,29 @@
 
 public class InstanceStoreMetadata {
 
-    private Map<String, Integer> instaceStoreCountMap = new HashMap<>();
+    private Map<String, VolumeParameterConfig> instaceStoreConfigMap = new HashMap<>();
 
     public InstanceStoreMetadata() {
     }
 
-    public InstanceStoreMetadata(Map<String, Integer> instaceStoreCountMap) {
-        this.instaceStoreCountMap = instaceStoreCountMap;
+    public InstanceStoreMetadata(Map<String, VolumeParameterConfig> instaceStoreConfigMap) {
+        this.instaceStoreConfigMap = instaceStoreConfigMap;
```

Comment:
I would add a null check here, to avoid null issues when `getInstaceStoreConfigMap()` is called. I think we should initialise an empty map if constructor is called explicitly with ` null`.